services/httpservice/httpservice.go

```diff
@@ -67,12 +67,24 @@ func (h *HTTPServiceImpl) MakeTransport(trustURLs bool) http.RoundTripper {
 	}
 
 	allowIP := func(ip net.IP) bool {
```

Comment:
Are there tests for `allowIP` that we can update?

Reply:
I fear that currently for the `HTTPServiceImpl` we are lacking unit tests altogether, I've added some unit tests covering the support functions `IsOwnIP` and `IsReservedIP`, however, I think we should open a follow up ticket to look into adding a test suite for the service altogether (potentially as a help wanted?)